lib/roger_jshint/generator.rb

```diff
@@ -0,0 +1,44 @@
+require 'open-uri'
+
+require 'roger/cli/generate'
+require 'roger/cli/command'
+require 'roger/generators'
+
+module RogerJsHint
+  # Lint configuration retriever
+  class Generator < Roger::Generators::Base
+    include Thor::Actions
+    CONFIG_PATH = '.jshintrc'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.